Jenkinsfile

```diff
@@ -72,7 +72,7 @@ node() {
                 sh 'id'
                 //sh 'whoami'
                 sh 'pwd'
-                sh "mvn clean install -T2 -Pci -V -B -DcleanNode -ntp -DforkCount=3 -Dmaven.test.failure.ignore -s settings.xml -e -Dmaven.repo.local=/tmp/m2 -Dmaven.artifact.threads=30"
+                sh "mvn clean install -T2 -Pci -V -B -DcleanNode -ntp -DforkCount=3 -s settings.xml -e -Dmaven.repo.local=/tmp/m2 -Dmaven.artifact.threads=30"
```

Comment:
this will not change anything for ci.jenkins.io :) You have to change the provided script by ci.j.io.
look at the start of the file https://github.com/jenkinsci/blueocean-plugin/blob/1ce0c18265882733f3251dcad0b4af616da5a7c9/Jenkinsfile#L3

this Jenkinsfile was only by ci.blueocean.io